config/install/hdf5-config.cmake.in

```diff
@@ -41,6 +41,8 @@ set (${HDF5_PACKAGE_NAME}_BUILD_MODE       @HDF_CFG_NAME@)
 set (${HDF5_PACKAGE_NAME}_PROVIDES_FORTRAN     @HDF5_BUILD_FORTRAN@)
 set (${HDF5_PACKAGE_NAME}_PROVIDES_CPP_LIB     @HDF5_BUILD_CPP_LIB@)
 set (${HDF5_PACKAGE_NAME}_PROVIDES_JAVA        @HDF5_BUILD_JAVA@)
+set (${HDF5_PACKAGE_NAME}_PROVIDES_JNI         @HDF5_ENABLE_JNI@)
+set (${HDF5_PACKAGE_NAME}_PROVIDES_COMPAT      ON) # Always provide the Java compatibility API
```

Comment:
I would suggest renaming this variable as well to something like `${HDF5_PACKAGE_NAME}_PROVIDES_JAVA_COMPAT`. `PROVIDES_COMPAT` alone doesn't tell you what the compat part is for at all.